webapp/sources/rudder/rudder-web/src/main/elm/sources/Editor.elm

```diff
@@ -448,11 +450,8 @@ update msg model =
       let
         action = case model.mode of
                    TechniqueDetails t _ _ ->
-                     let
-                       data =  encodeExportTechnique t
-                       content = Json.Encode.encode 2 data
-                     in
-                       File.Download.string (t.id.value ++ ".json") "application/json" content
+                       getTechniqueYaml model t
```

Comment:
this does not seems to work: I see the remote call being done, if I do it by hand I get the yaml, but with the button it does not open a download windows in the browser. The remote call result is just ignored.